src/App/Resources/AppResources.fr.resx

```diff
@@ -185,6 +185,22 @@
   <data name="CozyHomeHaveAccount" xml:space="preserve">
     <value>J'ai déjà mon Cozy</value>
   </data>
+  <data name="CozyUrlRequiredException" xml:space="preserve">
+    <value>L'URL du Cozy est requise</value>
```

Comment:
In several places we use `l'adresse` instead of `URL` see https://github.com/cozy/cozy-pass-mobile/blob/60520fccdc86cbd907ee1b666f593e6192613e3d/src/App/Resources/AppResources.fr.resx#L554-L556 for instance.

I find this word more user friendly. URL is "technical"

Reply:
Ok for using `adresse`. Should I do the same for english texts? I don't know if they are more used to `URL` or to `address`

Reply:
I don't know. What are we currently using? Also both?

Reply:
Yep... also both 😄

Reply:
I find usage of address in the UI so I will adapt the error messages

Reply:
Fixed